homeassistant/components/fan/services.yaml

```diff
@@ -120,8 +120,76 @@ xiaomi_miio_set_led_brightness:
   description: Set the led brightness.
   fields:
     entity_id:
-      description: Name of the air purifier entity.
-      example: 'fan.xiaomi_air_purifier'
+      description: Name of the xiaomi miio entity.
+      example: 'fan.xiaomi_miio_device'
     brightness:
       description: Brightness (0 = Bright, 1 = Dim, 2 = Off)
       example: 1
+
+xiaomi_miio_set_auto_detect_on:
+  description: Turn the auto detect on.
```

Comment:
Auto detect and brightness are relevant?

Reply:
Fixed.